test/fixtures/qunit/valid.js

```diff
@@ -2,25 +2,31 @@ QUnit.module( 'Example' );
 
 // Off: qunit/no-arrow-tests
 // Valid: qunit/require-expect
-QUnit.test( '.foo()', ( assert ) => {
+QUnit.test( '.foo()', () => {
+} );
+
+QUnit.test( '.bar()', function ( assert ) {
```

Comment:
I moved most of our code out the arrow function, because most eslint-plugin-qunit rules are broken in arrow functions (and always have been), see https://github.com/platinumazure/eslint-plugin-qunit/issues/213